docs/csharp/tutorials/pattern-matching.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Use pattern matching features to extend data types
+title: "Tutorial: pattern matching to return a value"
 description: This advanced tutorial demonstrates how to use pattern matching techniques to create functionality using data and algorithms that are created separately.
 ms.date: 03/13/2019
-ms-technology: csharp-whats-new
-ms.custom: mvc
+ms.technology: csharp-whats-new
+ms.custom: contperfq1
 ---
 # Tutorial: Using pattern matching features to extend data types
```

Comment:
Here again, I'm starting to like the focus on "data driven algorithms":

```suggestion
# Tutorial: Use pattern matching to build type-driven and data-driven algorithms.
```

Reply:
Good catch! I don't know how I missed the H1